back/engines/free/email_campaigns/app/models/email_campaigns/campaign.rb

```diff
@@ -29,6 +31,7 @@
 module EmailCampaigns
   class Campaign < ApplicationRecord
     belongs_to :author, class_name: 'User', optional: true
+    belongs_to :project, optional: true
```

Comment:
This association seems a bit out of place. Campaigns in general are not associated to a project, only one specific campaign.